runtime/ts/storage/crdt-collection-model.ts

```diff
@@ -43,6 +43,7 @@ export class CrdtCollectionModel {
   // or `value` is different to the value previously stored).
   add(id: string, value, keys): boolean {
     assert(keys.length > 0, 'add requires keys');
+    assert(typeof keys === 'object', 'add requires a list of keys');
```

Comment:
Why is this required? Does the first assert not suffice?

Reply:
this might just be a merge duplication? Not sure that the second assert does add anything.

Reply:
as you demonstrated to me, it's so that single strings aren't accepted (these also have a length).